iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/clients/clientslist/clientslist.component.html

```diff
@@ -105,6 +106,15 @@
                                 <!-- Dynamically registered -->
                                 {{ c.dynamically_registered }}
                             </td>
+                            <td class="col-xs-1">
```

Comment:
Hide this column in case IAM_CLIENT_TRACK_LAST_USED is false

Reply:
Done.